src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationTest.java

```diff
@@ -292,12 +292,10 @@ void validateRetrievedUsers(Map<String, Object> searchResponse, String expectedS
             if (!expectedStatus.equals("any")) {
                 assertThat(user.get("idamStatus").equals(expectedStatus));
             }
-            if (rolesReturned) {
-                if (user.get("idamStatus").equals(IdamStatus.ACTIVE.toString())) {
-                    assertThat(user.get("roles")).isNotNull();
-                } else {
-                    assertThat(user.get("roles")).isNull();
-                }
+            if (user.get("idamStatus").equals(IdamStatus.ACTIVE.toString()) && rolesReturned) {
+                assertThat(user.get("roles")).isNotNull();
+            } else {
+                assertThat(user.get("roles")).isNull();

```

Comment:
I believe this is the latest:

```
if (rolesReturned) {
                if (user.get("idamStatus").equals(IdamStatus.ACTIVE.toString())) {
                    assertThat(user.get("roles")).isNotNull();
                } else {
                    assertThat(user.get("roles")).isNull();
                }
            }
```

as per: https://github.com/hmcts/rd-professional-api/pull/611/files which was merged into Demo

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/612/commits/acbe4881fa2e5d46be921bc81e0a93e8f8bf9f4f